Robot/src/main/java/com/swrobotics/robot/commands/AutoAimCommand.java

```diff
@@ -0,0 +1,44 @@
+package com.swrobotics.robot.commands;
+
+import com.swrobotics.robot.subsystems.drive.DrivetrainSubsystem;
+import com.swrobotics.robot.subsystems.vision.Limelight;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class AutoAimCommand extends CommandBase {
+    private final Limelight limelight;
+    private final DrivetrainSubsystem drivetrainSubsystem;
+
+    Rotation2d limelight_target_angle;
+    Pose2d drivetrain_pose;
+
+    Rotation2d robot_target_angle;
+
+    public AutoAimCommand(Limelight limelight, DrivetrainSubsystem drivetrainSubsystem) {
+        this.limelight = limelight;
+        this.drivetrainSubsystem = drivetrainSubsystem;
+
+        addRequirements(limelight, drivetrainSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        // Get System State at Command Initialization to Ensure equal state
+        limelight_target_angle = limelight.getXAngle();
+        drivetrain_pose = drivetrainSubsystem.getPose();
+        robot_target_angle = drivetrain_pose.getRotation().plus(limelight_target_angle);
```

Comment:
You may want to continuously update the target angle in execute() since the angle would change if the robot gets bumped